runtime/lua/vim/_watch.lua

```diff
@@ -283,6 +283,15 @@ function M.fswatch(path, opts, callback)
     '/.git/',
     path,
   }, {
+    stderr = function(err, data)
+      if err then
+        error(err)
+      end
+
+      if data and #vim.trim(data) > 0 then
+        vim.notify(data, vim.log.levels.ERROR)
```

Comment:
```suggestion
        vim.schedule(function ()
          vim.notify(data, vim.log.levels.ERROR, {title = 'LSP Watch'})
        end)
```

Reply:
because of `vim/_editor.lua:0: E5560: nvim_err_writeln must not be called in a lua loop callback`.

Reply:
I initially tested it with Noice I initially tested it with Noice, so there was no such error before.